master/reference/calicoctl/commands/node/run.md

```diff
@@ -78,7 +78,7 @@ Options:
                            [default: /var/log/calico]
      --node-image=<DOCKER_IMAGE_NAME>
                            Docker image to use for Calico's per-node container.
-                           [default: quay.io/calico/node:%s]
+                           [default: quay.io/calico/node:latest]
```

Comment:
nope - this is hardcoded as latest now - because of the calico/node split.

Ditto below.